dockstore-webservice/src/main/java/io/dockstore/webservice/resources/LambdaEventResource.java

```diff
@@ -72,6 +73,24 @@ public List<LambdaEvent> getLambdaEventsByOrganization(@ApiParam(hidden = true)
         return lambdaEventDAO.findByOrganization(organization, offset, limit, authorizedRepos);
     }
 
+    @GET
+    @Timed
+    @UnitOfWork(readOnly = true)
+    @RolesAllowed({ "admin", "curator"})
+    @Path("/user/{userid}")
+    @Operation(operationId = "getUserLambdaEvents", description = "Get all of the Lambda Events for the given user.",
+            security = @SecurityRequirement(name = ResourceConstants.JWT_SECURITY_DEFINITION_NAME))
+    public List<LambdaEvent> getUserLambdaEvents(@Parameter(hidden = true, name = "user")@Auth User authUser,
+           @PathParam("userid") long userid,
+           @QueryParam("offset") @DefaultValue("0") Integer offset,
+           @DefaultValue("1000") @QueryParam("limit") Integer limit) {
```

Comment:
To improve readability, might as well use the same order for the annotations on each line.